package.json

```diff
@@ -29,17 +28,22 @@
     "@babel/core": "^7.13.15",
     "@babel/preset-env": "^7.13.15",
     "@babel/preset-react": "^7.13.13",
+    "@types/node": "^14.14.41",
     "babel-core": "^6.26.3",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
     "babel-runtime": "^6.26.0",
     "rollup": "^2.40.0",
+    "rollup-plugin-paths": "^0.0.4",
     "rollup-plugin-sass": "^1.2.2",
     "rollup-plugin-typescript2": "^0.30.0"
   },
+  "peerDependencies": {
+    "react": "^17.0.2",
```

Comment:
why this 17.0.2 instead of 17.0.1 ?

Reply:
It's just, I was trying to update react version to resolve the conflict before, I got the correct fixed to this issue.
I guess, now even if the version will be 17.0.1 thing will go smoothly..!!